MantidPlot/mantidplot.py

```diff
@@ -14,6 +14,27 @@
                         print_function)
 
 import os.path as osp
+import sys
```

Comment:
Yes, because we're not running through setuptools, the `.egg-links` are not expanded, and MantidPlot fails to initialise Python.

Looking at the `PYTHONPATH` after startup it seems that the `easy-install.pth` was actually read at some point, as it has added the other 2 paths from the `easy-install.pth`, but skipped the one I add with the egg-link.

I am wondering though why the `easy-install.pth` wasn't expanded at the point of `import mantid`? Otherwise, that also has `Framework/PythonInterface` in it. If we can make it expand that, I wouldn't have to open the `egg-link`.

Reply:
Actually I see why, the working directory is `'C:\\Users\\qbr77747\\dev\\m\\source\\build\\MantidPlot'`, which doesn't contain anything. Do we have a way of changing that?

Reply:
Resolved with https://github.com/mantidproject/mantid/pull/25385/commits/83e37fcbf318358a0c8bc57fa4ca367ff4283653, `site.addsitedir` is now done with the `bin` dir, which expands `bin/easy-install.pth`, and adds `Framework/PythonInterface`